lib/src/Uncertainty/Distribution/Multinomial.cxx

```diff
@@ -477,9 +477,10 @@ Sample Multinomial::getSupport() const
 {
   const UnsignedInteger dimension = getDimension();
   LinearEnumerateFunction enumerate(dimension);
-  const UnsignedInteger size = enumerate.getStrataCumulatedCardinal(n_);
+  const UnsignedInteger firstIndex = n_ == 0 ? 0 : enumerate.getStrataCumulatedCardinal(n_ - 1);
+  const UnsignedInteger lastIndex = enumerate.getStrataCumulatedCardinal(n_);
   Sample support(0, dimension);
-  for (UnsignedInteger i = 0; i < size; ++i)
+  for (UnsignedInteger i = firstIndex; i < lastIndex; ++i)
```

Comment:
@dbarbier There is a getStrataCardinal() to get the number of points in a given strata, no need to get it as a difference between two cumulated cardinals. BTW, there was no bug here, re-read the definition of the Multinomial class here:
http://openturns.github.io/openturns/master/user_manual/_generated/openturns.Multinomial.html?highlight=multinomial
In OpenTURNS we use a definition which reduces to the definition of the Binomial class when dimension=1. The classical definition leads to a singular discrete distribution, which is not a good thing. The classical definition is recovered by taking the last value of p equal to 0.
I agree that in this case, the support should be reduced (only the 'generic' support is implemented), but this change has also to be made into the Binomial and Bernoulli classes.